src/openforms/variables/constants.py

```diff
@@ -54,3 +56,17 @@ class DataMappingTypes(models.TextChoices):
     FormVariableDataTypes.date: check_date,
     FormVariableDataTypes.time: check_time,
 }
+
+
+# TODO-4980: move to openforms.utils.json_schema?
```

Comment:
Fine to keep it here - it's tightly coupled to the `FormVariableDataTypes` constant so it makes sense to keep that close together.